toast-tk-runtime/src/main/java/io/toast/tk/runtime/AbstractTestPlanRunner.java

```diff
@@ -128,39 +129,45 @@ public void execute(final ITestPlan project, final boolean presetRepoFromWebApp)
 			final ITestPage repoAsTestPageForConvenience = parser.readString(repoWiki, null);
 			runner.run(repoAsTestPageForConvenience);
 		}
-		execute(project, runner);
+		execute(testPlan, runner);
 	}
 
-	private void execute(final ITestPlan project, final TestRunner runner) {
+	private void execute(final ITestPlan testPlan, final TestRunner runner) {
 		initEnvironment();
-		for (final ICampaign campaign : project.getCampaigns()) {
+		for (final ICampaign campaign : testPlan.getCampaigns()) {
 			for (ITestPage testPage : campaign.getTestCases()) {
 				try {
 					beginTest();
-					testPage = runner.run(testPage);
+					runner.run(testPage);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "testPage". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1854)